### DISCUSSION TITLE
Turn Off console.log of Lit js SDK

### DISCUSSION THREAD

**Discussion by vasjak** [lit/lit#3989] [General]
Hi,

how could I turn off the console.logs of the Lit js SDK to use in production?
![image](https://github.com/lit/lit/assets/82120055/619b0ea2-ad26-4344-9575-bbb1a5929a78)

Thanks,
Vasja

**Comment by VandeurenGlenn** [comment]
@vasjak 
This is not the Lit you searching for

**Reply by vasjak** [reply]
sorry, thanks
